hack/tools/go.mod

```diff
@@ -4,8 +4,8 @@ go 1.20
 
 require (
 	github.com/jteeuwen/go-bindata v3.0.7+incompatible
-	k8s.io/code-generator v0.26.1
-	sigs.k8s.io/controller-tools v0.11.4
+	k8s.io/code-generator v0.27.1
```

Comment:
you have bumped other k8s packages to v0.27.2 in root go.mod and api/go.mod. Can you check if we can do the same for this package.